deepfence_server/reporters/scan/scan_result_actions.go

```diff
@@ -223,7 +224,7 @@ func DeleteScan(ctx context.Context, scanType utils.Neo4jScanType, scanId string
 	return nil
 }
 
-func StopCloudComplianceScan(ctx context.Context, scanType, scanId string) error {
+func StopCloudComplianceScan(ctx context.Context, scanType string, scanIds []string) error {
```

Comment:
nit: Let's remove `scanType` entirely since it's not used

Reply:
Removed.